lib/postinstall.js

```diff
@@ -38,9 +38,9 @@ ${content}`);
 const writePrettierRc = (new Promise((resolve, reject) => {
   const prettierPath = path.join(path.resolve('..', '..'), '.prettierrc');
   const content = `{
```

Comment:
You can just use `JSON.stringify(content, null, '  ')` or `JSON.stringify(content, null, '\t')`. 

(Not an option for the other one because it's supposed to be JavaScript 🙄 )